webapp/channels/src/components/admin_console/admin_definition.jsx

```diff
@@ -2447,6 +2447,17 @@ const AdminDefinition = {
                             it.licensedForSku(LicenseSkus.Professional),
                         )),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'ServiceSettings.RefreshPostStatsRunTime',
```

Comment:
I'm not entirely sure where the setting for this actually belongs, so for now I've put it under `Site Configuration -> Users and Teams` and it's in the `ServiceSettings` object.

I wasn't sure if we wanted a different screen for any of the new reporting settings, and potentially a new section for it as well?

Reply:
@devinbinnie What other settings would we have that could go in this new section for `Reporting Settings`? We could have settings for exports but I'm thinking maybe we can revisit this when we add those and let this one be where you have it for now?